WORKSPACE

```diff
@@ -28,9 +28,9 @@ load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
 
 rules_js_dependencies()
 
-load("@aspect_rules_js//js:toolchains.bzl", "DEFAULT_NODE_VERSION", "rules_js_register_toolchains")
+load("@aspect_rules_js//js:toolchains.bzl", "rules_js_register_toolchains")
 
-rules_js_register_toolchains(node_version = DEFAULT_NODE_VERSION)
+rules_js_register_toolchains(node_version = "20.11.0")
```

Comment:
The v20 now needs a minimum of 20.19 but may be better for performance to use 22.12.0